src/Nav.jsx

```diff
@@ -7,45 +7,74 @@ const Nav = () => {
       justifyContent="space-between"
       mt="10px"
       bg="gray.700"
-      borderRadius="10px"
+      borderRadius="30px"
       mx="10px"
       py="20px"
       sx={{
         ".active": { fontWeight: "bolder" },
       }}
+      mb="10px"
     >
       <Box display="flex" ml="20px">
         <Box
           _hover={{
-            color: "lightblue",
+            color: "rgb(111, 111, 182)",
             transition: "color 0.2s ease-out",
           }}
-          pr="10px"
+          pr="30px"
         >
           <NavLink
             className={({ isActive }) => (isActive ? "active" : "")}
-            to="links"
+            to="preview/links"
           >
             Links
           </NavLink>
         </Box>
```

Comment:
Create a reusable component that will handle the link styles and path. So you can just pass something like:
<AppLink path='preview/links' label="Links" />